restaking_core/src/config.rs

```diff
@@ -49,9 +50,8 @@ impl Config {
         }
     }
 
-    pub fn get_epoch_from_slot(&self, slot: u64) -> Result<u64, RestakingError> {
-        slot.checked_div(self.epoch_length())
-            .ok_or(RestakingError::InvalidEpochLength)
+    pub fn get_epoch_from_slot(&self, slot: u64) -> Result<u64, CoreError> {
```

Comment:
same here, remove this fn + use get_epoch everywhere

Reply:
Done